src/sandstorm/supervisor.capnp

```diff
@@ -107,6 +107,10 @@ interface SystemPersistent extends(Persistent(Data, ApiTokenOwner)) {
   # created for the purpose of HTTP APIs).
 }
 
+interface HandlePersistent extends(SystemPersistent, Util.Handle) {}
```

Comment:
I've generally used `Persistent` as a prefix rather than a suffix for these cases. (`SystemPersistent` is different because it's not "a persistent System", it's "the system's Persistent interface".)